### PR TITLE
Adds air alarm chirping.

### PR DESCRIPTION
**Pull request by ghost**
It happens every 4-5 minutes after the power goes out, but if the power goes back on it will reset the countdown.

:cl:
 * rscadd: Air alarms now chirp while the power is out.

### CODE REVIEW

**Comment by pedr69** [comment]
i say remove the changelogs
see how long it takes for people to notice

File: code/game/machinery/alarm.dm [hunk 1]
```diff
@@ -351,8 +353,15 @@ var/global/list/air_alarms = list()
 /obj/machinery/alarm/process()
 	if((stat & (NOPOWER|BROKEN|FORCEDISABLE)) || shorted || buildstage != 2)
```
Comment (boy2mantwicethefam) [inline code/game/machinery/alarm.dm]: This makes the sound play even if the machinery is outright broken, manually disabled or under a stage of construction.

Comment (ghost) [reply code/game/machinery/alarm.dm]: You're no fun.

**Comment by Securitymain** [comment]
This is an april 1st thing

actually when is black history month?

**Comment by Inorien** [comment]
>have fire alarms

>puts it on air alarm

**Comment by Pastamadness** [comment]
Meme PR

**Comment by Kurfursten** [comment]
It's funny, but it doesn't work the way you think it does (I don't believe you tested this). Air alarms will not be NOPOWER unless environment power fails, which is the absolute last priority behind lights and equipment. In other words, there will be no chirping unless the room is so low on power that its airlocks are offline. I also agree this makes more sense on fire alarms, but more generally we don't really need a sound notification for this.

**Pull request closed without merging**